documentation/connect_puppet_master.markdown

```diff
@@ -118,7 +118,17 @@ Create it if necessary, and add the following:
         terminus: puppetdb
         cache: yaml
 
-## Step 3: Restart Puppet Master
+## Step 3: Set Security Policy
+
+PuppetDB listens on tcp ports 8080 (http) and 8081 (https). Ensure these ports
```

Comment:
We don't support talking to HTTP from the master, so this isn't entirely true or necessary to open up 8080. The only port that requires public access is generally 8081, unless someone is doing something special.